storage/client_range_test.go

```diff
@@ -150,3 +154,156 @@ func TestRejectFutureCommand(t *testing.T) {
 		t.Errorf("expected 15, got %v", v)
 	}
 }
+
+// Test a case where a put operation of an older timestamp comes after
```

Comment:
nit: comment should take the form `// TestOutOfOrderPut ...`. We don't lint the tests, but let's stick with it for uniformity.